### PR TITLE
rework a bit the grid layout to accomodate the footer

### PR DESCRIPTION
**Pull request by vujevits**
### Fixes
- page is scrollable
- resizing the window does not messes up the footer

### CODE REVIEW

File: packages/app/src/App.tsx [hunk 1]
```diff
@@ -1,29 +1,29 @@
...
+import { Box, Grid } from "grommet";
 import { BrowserRouter as Router } from "react-router-dom";
 
-import TestnetBanner from "./components/testnet-banner";
 import Home from "./pages/home";
 import Token from "./pages/token";
 import { NotificationsList } from "./components/transactions";
 import Foot from "./components/footer";
-import { Grid } from "grommet";
+import Navbar from "./components/nav";
```
Comment (kyriediculous) [inline packages/app/src/App.tsx:10]: The navbar doesn't need to be on the landing page only in the app

File: packages/app/src/App.tsx [hunk 2]
```diff
@@ -1,29 +1,29 @@
...
 
 const App: FC = () => {
   return (
-    <>
-      <Grid fill rows={["xxsmall", "flex", "xsmall"]}>
-        <TestnetBanner />
-        <Router>
+    <Router>
+      <Grid height="100vh" rows={["xsmall", "1fr"]}>
```
Comment (kyriediculous) [inline packages/app/src/App.tsx:15]: The grid can be larger than 100vh

**Review by kyriediculous — CHANGES_REQUESTED** [review]

File: packages/app/src/App.tsx (hunk 2 above)
Comment (vujevits) [reply packages/app/src/App.tsx]: changed it to `min-height` to be more explicit, but it did have the same behavior (without it it does not fill the screen)

File: packages/app/src/App.tsx (hunk 1 above)
Comment (vujevits) [reply packages/app/src/App.tsx:10]: I moved the banner to the navbar, and we are only showing that on the main page

File: packages/app/src/App.tsx (hunk 2 above)
Comment (kyriediculous) [reply packages/app/src/App.tsx]: Right, but we also want to add additional content to the home page below the initial pagefold eventually so being more explicit here doesn't hurt I guess

**Review by kyriediculous — APPROVED** [review]

**Pull request merged**